src/traccuracy/loaders/_point.py

```diff
@@ -0,0 +1,90 @@
+from typing import Optional, cast
+
+import networkx as nx
+import pandas as pd
+
+from traccuracy._tracking_graph import TrackingGraph
+
+
+def load_point_data(
+    path: str | None = None,
+    df: Optional[pd.DataFrame] = None,  # noqa: UP007, bar syntax breaks docks build
+    parent_column: str = "parent",
+    id_column: str | None = None,
+    pos_columns: tuple[str, ...] = ("z", "y", "x"),
+    time_column: str = "t",
+    name: str | None = None,
+    sep: str | None = None,
+) -> TrackingGraph:
+    """Load point-based tracking data into a TrackingGraph from a csv-like file
+
+    Assumes each row contains:
+    - time
+    - position, e.g. three columns 'z', 'y', 'x'
+    - parent, a reference to the node in the previous time frame
```

Comment:
Document how to say there is no parent (-1? None? either?)

Reply:
Do you think I should make this a settable option?